pkg/models/models.go

```diff
@@ -336,6 +337,11 @@ type MalwareAnalysisResult struct {
 	VerificationRecord *malysisv1.VerificationRecord
 }
 
+type CodeAnalysisResult struct {
+	// Usage evidences of a package obtained by depsusage plugin
+	UsageEvidences []*ent.DepsUsageEvidence `json:"usage_evidences"`
```

Comment:
Hope `ent` models are JSON serializable. Do verify:

```
vet scan ... --json-dump-dir /tmp/dump
```

Open the JSON file inside the `/tmp/dump` and verify that usage evidences are available in the JSON. The reason why we don't need a specific test case here is JSON serialization is an `ent` concern. We just need to verify once

Reply:
yes, verified the JSON, also it has options for customisation eg. https://entgo.io/docs/schema-fields/#struct-tags